src/webclients/components/sdfv/utils/attributes_table.ts

```diff
@@ -1502,6 +1522,73 @@ export function appendDataDescriptorTable(
     });
 
     VSCodeSDFV.getInstance().getMetaDict().then(metaDict => {
+        const updateNameListener = (prop: KeyProperty) => {
+            // When a data container name is changed, update the data container
+            // and label for all access nodes referencing this data container.
+            const nVal = prop.getValue();
+            if (nVal.valueChanged) {
+                const oldDescriptor = prop.getKey();
+                const newDescriptor = nVal.value;
+
+                doForAllNodeTypes(
+                    sdfg, 'AccessNode', (accessNode: JsonSDFGNode) => {
```

Comment:
I hope this doesn't traverse recursively and change unrelated access nodes?

Reply:
The function would traverse recursively, but the last positional argument (`false`) tells it not to :)